pkg/client/restclient/versions.go

```diff
@@ -46,10 +46,11 @@ func ServerAPIVersions(c *Config) (groupVersions []string, err error) {
 	configCopy := *c
 	configCopy.GroupVersion = nil
 	configCopy.APIPath = ""
-	baseURL, _, err := defaultServerUrlFor(&configCopy)
+	baseURLs, _, err := defaultServerUrlFor(&configCopy)
 	if err != nil {
 		return nil, err
 	}
+	baseURL := baseURLs[0]
```

Comment:
When is version called?  We need a comment to justify that taking the first one is safe:

* If the first server is down
* If the servers have different versions